plugins/maciVoting/pages/proposal-list.tsx

```diff
@@ -36,6 +36,9 @@ export default function Proposals() {
     abi: MaciVotingAbi,
     functionName: "proposalCount",
     chainId: PUBLIC_CHAIN.id,
+    query: {
```

Comment:
I am wondering if this already fixes the `isTallied()` error? 🤔 

1. User submits a proposal
2. Gets redirected to the proposal list page
3. Data gets refetched with the newly deployed proposal

Reply:
Is kinda related, but need to double check on that, as for now i am not seeing the error anymore